tests/plugins/image/image.test.ts

```diff
@@ -0,0 +1,92 @@
+import { createImage, paste } from "../../test_helpers/commands_helpers";
+
+describe("image plugin tests", function () {
+  test("create image", () => {
+    const model = new Model();
+    const sheetId = model.getters.getActiveSheetId();
+    const imageId = "Image1";
+    const definition = {
+      sheetId,
+      path: "image path",
+      size: { width: 100, height: 100 },
+    };
+    createImage(model, { figureId: imageId, definition: definition });
+    expect(model.getters.getImage(imageId)).toEqual(definition);
+  });
+
+  test("delete image", () => {
+    const model = new Model();
+    const sheetId = model.getters.getActiveSheetId();
+    const imageId = "Image1";
+    createImage(model, { figureId: imageId, definition: { sheetId: sheetId } });
+    model.dispatch("DELETE_FIGURE", { sheetId, id: imageId });
+    const images = model.getters.getImageIds(sheetId);
+    expect(images).toHaveLength(0);
+  });
+
+  test("copy past image", () => {
+    const model = new Model();
+    const sheetId = model.getters.getActiveSheetId();
+    const imageId = "Image1";
+    const definition = {
+      sheetId,
+      path: "image path",
+      size: { width: 100, height: 100 },
+    };
+    createImage(model, { figureId: imageId, definition: definition });
+    model.dispatch("SELECT_FIGURE", { id: imageId });
+    model.dispatch("COPY");
+    paste(model, "D4");
+    const images = model.getters.getImageIds(sheetId);
+    expect(images).toHaveLength(2);
+    for (const nextImageId of images) {
+      expect(model.getters.getImage(nextImageId)).toEqual(definition);
+    }
+  });
+
+  test("cut past image", () => {
+    const model = new Model();
+    const sheetId = model.getters.getActiveSheetId();
+    const imageId = "Image1";
+    const definition = {
+      sheetId,
+      path: "image path",
+      size: { width: 100, height: 100 },
+    };
+    createImage(model, { figureId: imageId, definition: definition });
+    model.dispatch("SELECT_FIGURE", { id: imageId });
+    model.dispatch("CUT");
+    paste(model, "D4");
+    const images = model.getters.getImageIds(sheetId);
+    expect(images).toHaveLength(1);
+    const image = model.getters.getImage(images[0]);
+    expect(image).toEqual(definition);
+  });
+
+  test("duplicate sheet image", () => {
+    const model = new Model();
+    const sheetId = model.getters.getActiveSheetId();
+    const imageId = "Image1";
+    createImage(model, { figureId: imageId, definition: { sheetId: sheetId } });
+    const newSheetId = "Sheet2";
+    model.dispatch("DUPLICATE_SHEET", { sheetId, sheetIdTo: newSheetId });
+    const original = model.getters.getImage(imageId);
+    const newImages = model.getters.getImageIds(newSheetId);
+    expect(newImages).toHaveLength(1);
+    const copy = model.getters.getImage(newImages[0]);
+    expect(copy.sheetId).not.toBe(original.sheetId);
+    expect(copy.sheetId).toBe(newSheetId);
+  });
+
+  test("delete sheet image", () => {
```

Comment:
```suggestion
  test("delete a sheet with an image", () => {
```
I though it was a duplicate of `delete image` at first glance